src/azure-cli/azure/cli/command_modules/vm/_params.py

```diff
@@ -1273,7 +1275,7 @@ def load_arguments(self, _):
 
     for scope in ['sig image-version create', 'sig image-version update']:
         with self.argument_context(scope) as c:
-            c.argument('target_regions', nargs='*', validator=process_gallery_image_version_namespace,
```

Comment:
Since command `sig image-version update` also called `validator=process_gallery_image_version_namespace`, we may add a command level validator for command `sig image-version update`, and then call `process_gallery_image_version_namespace` in the command level validator